README.md

```diff
@@ -0,0 +1,24 @@
+# Bot Deployed By Nethermind Deployer Bot
+
+## Description
+
+This bot detects creations of forta bots by the forta deployer address.
+
+## Supported Chains
+
+- Polygon
+
+## Alerts
+
+- Forta-Bot-Deployed
+  - Fired when the function `createAgent` is called on the Forta contract
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata contains:
+    - Provides the deployer, agentId and chainIds
```

Comment:
```suggestion
      - `agentId`: agentId of the deployed bot
      - `metadata`: metadata of the deployed bot
      - `owner`: deployer address
      - `chainIds`: supported chains of the bot
```

this should be the structure here.
can you also add these fields in your finding object?